config.txt

```diff
@@ -1,9 +1,15 @@
-core_freq=400
-dtoverlay=spi0-1cs
+gpu_mem=32
 dtparam=i2c_arm=on
+dtparam=i2s=on
 dtparam=spi=on
 dtparam=audio=on
-avoid_warnings=1
+dtoverlay=i2c-gpio
+dtoverlay=dwc2,dr_mode=host
+dtoverlay=spi0-1cs
+dtoverlay=spi1-3cs
```

Comment:
Does it use multiple SPI buses?

Reply:
No, it does not, but some reason that I don't remember right now, I had to add the second SPI bus at some point in the current OS. So unless there's good reason to actually remove it, I'd keep it :)